iceoryx_hoofs/test/moduletests/test_cxx_function_ref.cpp

```diff
@@ -166,8 +166,11 @@ TEST_F(function_refDeathTest, CallMovedFromLeadsToTermination)
     function_ref<int()> sut1{lambda};
     function_ref<int()> sut2{std::move(sut1)};
     // Use after move is tested here
-    // NOLINTNEXTLINE (bugprone-use-after-move)
+    // NOLINTBEGIN(bugprone-use-after-move, hicpp-invalid-access-moved, cppcoreguidelines-pro-type-vararg,
```

Comment:
Does multiline `NOLINTBEGIN` work? Is this ignored when you run `clang-tidy` manually?

Reply:
@elfenpiff Unfortunately, I'm still running clang 13 locally, which does not support this syntax. However, the clang15 in the CI seems to run fine with.

Reply:
Works also locally for me.